localisation/00_zuwc_united_localisation.csv

```diff
@@ -3305,3 +3305,46 @@ awaken_paragon_desc;Ask the Klaxxi to awaken [Root.GetBestName].;;Bittet die Kla
+EVTTITLE_WCVOD_10110;The Price of Defiance;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10110;After you have disposed of your 'benefactors', you began preparing yourself for the inevitable consolidation of power within your realm. However, the Twilight's Hammer didn't forget about your betrayal... Multiple shadowy figures converge upon your court, and now you realize that you have to fight for your life in order to keep your prized stone body. If the visions were correct, you should prevail and emerge as the sole uncontested ruler of the reborn Neferset.;;;;;;;;;;;;;x
+EVTTITLE_WCVOD_10120;The Dark Pharaoh Rises;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10120;You fought with everything you had at your disposal, and ultimately emerge victorious from the hectic clash against your former would-be allies. Soon enough, you manage to compose yourself and proclaim a new era for the Neferset kingdom. Many of your subjects look upon you with mixture of confusion, shock and awe. You return to your throne room and prepare your realm for the inevitable confrontation with the others.;;;;;;;;;;;;;x
+EVTTITLE_WCVOD_10121;The Fall;;;;;;;;;;;;;x
+EVTDESC_WCVOD_10121;You fought with everything you had at your disposal, but it was not enough. You become overwhelmed by the barrage of shadow missiles cast by the Twilight priests, and soon enough you're bested in combat. You manage to land a final glance upon your executioner, who does not hesitate to cast one final bolt of shadow magic to ultimately end your life.;;;;;;;;;;;;;x
+defiance_wont_go_unpunished;�RDefying the Twilight Agents will come at a price...�!;;;;;;;;;;;;;x
+EVTOPTA_WCVOD_10100;The Curse of Flesh can be lifted? Tell me more.;;;;;;;;;;;;;x
+EVTOPTB_WCVOD_10100;Guards! Take these heathens away from here.;;;;;;;;;;;;;x
+EVTOPTA_WCVOD_10105;I shall do as the Old Gods command.;;;;;;;;;;;;;x
+EVTOPTB_WCVOD_10105;I do not serve. I rule.;;;;;;;;;;;;;x
+EVTOPTA_WCVOD_10110;Curses!;;;;;;;;;;;;;x
+EVTOPTA_WCVOD_10120;The power of the Void shall obey me.;;;;;;;;;;;;;x
+EVTOPTA_WCVOD_10121;This is not what the vision promised...;;;;;;;;;;;;;x
+EVTTITLE_WCCRS_210;A Great Stir in the East;;;;;;;;;;;;;x
+EVTDESC_WCCRS_210;Ever since the malevolent presence began tainting the landscape of Uldum with its foul magic, the ancient tombs have been stirring endlessly, as if responding to the corruption festering upon the sacred land of the Makers. In order to determine what exactly causes this phenomenon, you decide to investigate alongside a party of tol'vir adventurers.;;;;;;;;;;;;;x
+EVTTITLE_WCCRS_215;Tomb of the Sun King;;;;;;;;;;;;;x
+EVTDESC_WCCRS_215;As you and your group move through the hallways of the ancient tomb, you can't shake off the ominous feeling of dread and uncertainty. Perhaps going here was not a bright idea, after all... However, if the tombs have a key to get rid of the corruption, it would be for the best of Uldum. It's time to choose, go on or retreat and seal the tomb.;;;;;;;;;;;;;x
```

Comment:
Changed going to coming, and altered the third sentence a little to clean up some confusing phrasing.
```suggestion
EVTDESC_WCCRS_215;As you and your group move through the hallways of the ancient tomb, you can't shake off the ominous feeling of dread and uncertainty. Perhaps coming here was not a bright idea, after all... However, if the tombs have a key to get rid of the corruption, it would solve many of Uldum's problems. It's time to choose, go on or retreat and seal the tomb.;;;;;;;;;;;;;x
```